README.md

```diff
@@ -1053,6 +1053,7 @@ Utilities for managing game servers.
 - [Lancache](https://lancache.net) `⚠` - LAN Party game caching made easy. ([Source Code](https://github.com/lancachenet/monolithic)) `MIT` `Docker/Shell`
 - [LinuxGSM](https://linuxgsm.com/) - CLI tool for deployment and management of dedicated game servers on Linux: more than 120 games are supported. ([Source Code](https://github.com/GameServerManagers/LinuxGSM)) `MIT` `Shell`
 - [Lodestone](https://github.com/Lodestone-Team/lodestone) - A free, open source server hosting tool for Minecraft and other multiplayers. `AGPL-3.0` `Docker/Rust`
+- [Moonlight](https://moonlight-stream.github.io) - Remote game stream client for Sunshine with support up to 120 frames per second and 4K resolution. ([Source Code](https://github.com/moonlight-stream)) `GPL-3.0` `C++`
```

Comment:
This is the desktop/mobile client, right? W e only list web applications and network services.

Reply:
Correct, these are all the clients that Sunshine can connect to. It's not a host nor can you use it other than connecting for a host. Shall we mark this invalid?